cms/forms/validators.py

```diff
@@ -22,6 +24,20 @@ def validate_url(value):
         URLValidator()(value)
 
 
+def validate_url_extra(value):
+    try:
+        # Try normal validator first
+        validate_url(value)
+    except ValidationError:
+        # Fallback to absolute urls
+        if re.match(r'^tel:[0-9\+\#\*\-\.\(\)]+$', value):
```

Comment:
Validating phone numbers internationally is probably a total pain. Unless there is a standard way of doing this, I'd leave it at this.